src/EditorFeatures/CSharpTest/Completion/CompletionProviders/AwaitCompletionProviderTests.cs

```diff
@@ -0,0 +1,267 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Completion.Providers;
+using Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.Completion.CompletionProviders;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.Recommendations
+{
+    /// <summary>
+    /// The <see cref="AwaitCompletionProvider"/> adds async modifier if the return type is Task or ValueTask.
+    /// The tests here are only checking whether the completion item is provided or not.
+    /// Tests for checking adding async modifier are in:
+    /// src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests_AwaitCompletion.vb
+    /// </summary>
+    [Trait(Traits.Feature, Traits.Features.Completion)]
+    public class AwaitKeywordRecommenderTests : AbstractCSharpCompletionProviderTests
```

Comment:
We should also test things like commit, glyph and inline description here: tests in `CSharpCompletionCommandHandlerTests` are going through editor's async completion layer, so it'd worth to test those behavior here as "unit tests" too.

Reply:
> We should also test things like commit, glyph and inline description here

@genlu Any examples on the "commit" part from existing tests?